File: My project/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Pyramid height: ")
    #get me a pyramid between 1 to 8
    if height >=1 and height <= 8:
        break

#logic for pyramid
for i in range(height):
    for j in range(height - i - 1):
        print(" ", end = "")
    for k in range(-1 , i):
        print("#", end = "")
    print("  ", end = "")
    for l in range(-1, i):
        print("#", end = "")
    print()

File: My project/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do{

        height = get_int("height : ");

    }
    while(height < 1 || height > 8 );


for(int i=0; i<height; i++){

  for(int space =0; space < height-i-1 ; space++ ){

      printf(" ");
  }

        for(int j=0 ; j<=i ; j++){


            printf("#");

        }
        printf("\n");




    }

}






